internal/lib/constants.go

```diff
@@ -105,6 +105,8 @@ const (
 	DefaultRouteCert                           = "router-certs-default"
 	autoAnnotateService                        = "AUTO_ANNOTATE_SERVICE"
 	SeGroupLabelKey                            = "clustername"
+	UpdateOperation                            = "update"
```

Comment:
nit: could we use UpdateStatus/DeleteStatus instead?

Reply:
Done